dockyard/common/image/image.py

```diff
@@ -0,0 +1,34 @@
+from dockyard.common import base, link
+from dockyard.common import utils
+
+class Image(object):
+    def __init__(self):
+        self.rest_client = base.RESTClient()
+
+    def list(self, _id=None):
+        return utils.dispatch_get_request(protocol='http', url='/images/json').data
+
+    def history(self, _id=None):
+        url='/images/%s/history' % _id
+        return utils.dispatch_get_request(protocol='http', url=url).data
+
+    def search(self, term=None):
+        url='/images/search'
+        query = { "term" : term}
+        return utils.dispatch_get_request(protocol='http', url=url, query_params=query).data
+
+    def create(self, fromImage, tag):
```

Comment:
In the complete code written so far, there is not designing for the exceptions there we have to decide something about it.